pkgs/tools/misc/lorri/default.nix

```diff
@@ -12,10 +12,10 @@
 
 let
   # Run `eval $(nix-build -A lorri.updater)` after updating the revision!
-  version = "1.3.1";
-  gitRev = "df83b9b175fecc8ec8b02096c5cfe2db3d00b92e";
-  sha256 = "1df6p0b482vhymw3z7gimc441jr7aix9lhdbcm5wjvw9f276016f";
-  cargoSha256 = "1f9b2h3zakw7qmlnc4rqhxnw80sl5h4mj8cghr82iacxwqz499ql";
+  version = "1.4.0";
+  gitRev = "fee4ffac9ee16fc921d413789cc059b043f2db3d";
```

Comment:
Why is this not using the git tag for fetchFromGitHub.

Reply:
Idk. I thought that there maybe is some reason 🤷

Reply:
Lets wait a few days if a maintainer answers and if not the git tag should be fetched as this can't be a ref from a fork.

Reply:
`lorri` doesn't currently build and upgrading fixes that. Since this is mostly a stylistic issue, I think we can ignore it for now in preference of fixing the build.